src/remediations/controller.fifi.js

```diff
@@ -26,7 +26,18 @@ exports.connection_status = errors.async(async function (req, res) {
         throw new errors.Forbidden();
     }
 
-    const status = await fifi.getConnectionStatus(remediation, req.identity.account_number, req.entitlements.smart_management);
+    const rhcEnabled = await fifi.checkRhcEnabled();
```

Comment:
might be better to do this in parallel with the db query to speed up the operation

Reply:
Will do!  making the change now.